PhysicsTools/PatAlgos/python/slimming/slimmedLowPtElectrons_cfi.py

```diff
@@ -43,3 +43,9 @@
    )
 )
 
+from RecoEgamma.EgammaTools.lowPtElectronModifier_cfi import lowPtElectronModifier
+from Configuration.Eras.Modifier_run2_miniAOD_devel_cff import run2_miniAOD_devel
+from Configuration.Eras.Modifier_bParking_cff import bParking
+_slimmedLowPtElectrons = slimmedLowPtElectrons.clone()
+_slimmedLowPtElectrons.modifierConfig.modifications += [lowPtElectronModifier]
+(~bParking & run2_miniAOD_devel).toReplaceWith(slimmedLowPtElectrons,_slimmedLowPtElectrons)
```

Comment:
@slava77 the dxy and dz vars, plus the conversions vars, are only intended to be added for future re-miniAOD campaigns.